### ISSUE TITLE
The "HeuristicLab.ExtLibs.sln" and "HeuristicLab 3.3.sln" could not be built!

### ISSUE THREAD

**Issue by Zhixing1020** [t-h-e/HeuristicLab.CFGGP#4]
I planned to implement the G3GP for program synthesis problems and I followed the README in HeuristicLab.CFGGP to clone the HeuristicLab from Github. However, the "HeuristicLab.ExtLibs.sln" and "HeuristicLab 3.3.sln" in cloned repo couldn't work.  At first, I guessed the problem is the mismatch version of .Net framework. But after I upgraded my .Net framework of my Visual Studio, there were still so many building errors.

How can I build the repo successfully, can we give out more details about building these two .sln files on the README?

**Comment by Zhixing1020** [comment]
It is also advisable there can are more simple examples for users.

**Comment by t-h-e** [comment]
Hi, great to hear that you want to work on program synthesis.
Could you try https://github.com/t-h-e/HeuristicLab instead of the HeuristicLab referenced in the README? The version is different (3.3.14) vs (3.3.16)
I will update this repo at the next chance I get and improve the README in general.

Could you also give me some info about what you are using to build (OS, .Net framework, Visual Studio...) and the corresponding versions?

**Comment by t-h-e** [comment]
I just tested building with HeuristicLab (3.3.14) from a windows command line and it worked:
```
git clone https://github.com/t-h-e/HeuristicLab.git

msbuild /p:Configuration=Release /p:Platform=x64 HeuristicLab\HeuristicLab.ExtLibs.sln
msbuild /p:Configuration=Release /p:Platform=x64 "HeuristicLab\HeuristicLab 3.3.sln"

git clone https://github.com/t-h-e/HeuristicLab.CFGGP.git

msbuild /p:Configuration=Release /p:Platform=x64 HeuristicLab.CFGGP\HeuristicLab.CFGGP.sln

"HeuristicLab\bin\HeuristicLab 3.3.exe"
```

You can find msbuild in your Visual Studio directory (e.g.: `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin`) or get it from [here.](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild?view=vs-2019)

You can find G3GP/CFG-GP when you create a new item under "Problems" -> "Genetic Programming" -> "CFG Python Problem".
There is also "Context Free Grammar Problem" available for any non Python related grammar problem.

**Comment by Zhixing1020** [comment]
Hi，I am very grateful to tell you that version 3.3.14 did work! Successful for all these three .sln files!
I also want to share some problems I met in version 3.3.16. My OS is window10, .Net framework is 4.6.1 (it used to be 4.5.1), using VS2015. When I built the ExtLibs.sln, it reported the file "HeuristicLab\HeuristicLab.ExtLibs\HeuristicLab.OrTools\7.0.0\HeuristicLab.OrTools-7.0.0\Google.OrTools.pdb” was missing and it did be not in the original folder. After I removed this .pdb file from the solution, the building is correct. Whereas, the "HeuristicLab 3.3.sln" could not be built reporting more than 10 thousand errors. I guessed the remove of that .pdb file made some binary files would not be built, which made some class linkings were failure.

Some more questions
Since I am trying to re-run this project, how can I change some other hyper-parameters such as the iteration number (i.e. the max generation of GP) in the files? And how to use the HeuristicLab 3.3.exe? (I built a new item in the software but don't know how to excute the item)
